data/shared/citizen/scripting/v8/main.js

```diff
@@ -396,10 +396,12 @@ const EXT_LOCALFUNCREF = 11;
 					global.source = null;
 					return;
 				}
-
+        
 				global.source = parseInt(source.substr(4));
+			} else if (IsDuplicityVersion() && source.startsWith('internal-net')) {
+				global.source = parseInt(source.substr(13));
 			}
-
+			
```

Comment:
No idea why this indenting stuff is still broken, I *just* resolved the merge conflict and now these random tabs and spaces just above have just appeared :(